scripts/example.ts

```diff
@@ -0,0 +1,119 @@
+import path from 'path';
+import { existsSync } from 'fs';
+
+import { getOptionsOrPrompt } from './utils/options';
+import type { CLIStep } from './utils/cli-step';
+import { executeCLIStep } from './utils/cli-step';
+
+const frameworks = ['react'];
+const addons = ['a11y', 'storysource'];
+
+async function getOptions() {
+  return getOptionsOrPrompt('yarn example', {
+    framework: {
+      description: 'Which framework would you like to use?',
+      values: frameworks,
+      required: true,
+    },
+    addon: {
+      description: 'Which extra addons (beyond the CLI defaults) would you like installed?',
+      values: addons,
+      multiple: true,
+    },
+    includeStories: {
+      description:
+        "Include Storybook's own stories (only applies if a react-based framework is used)?",
+    },
+    create: {
+      description: 'Create the example from scratch (rather than degitting it)?',
+    },
```

Comment:
Perhaps we should change this to take the `before` folder from the template (before storybook is initialized on top of it).
And then run our local version of the CLI on it.
There really is no point in ever running the command to generate the `before` locally..

I see the point of bootstrapping with the local CLI for development though.

I propose:
```suggestion
    initStorybookLocally: {
      description: 'Degit the raw un-initialized template, and initialize storybook with the local CLI,
    },
```

Reply:
I see. Well I think we can start hammering out these details. I was really just trying to illustrate the structure of how it should work at this stage. I think ideally each step that the script run is itself a CLI command or another script, that would make it easier to run the steps piecemeal I think.